llvm/test/CodeGen/AArch64/trampoline.ll

```diff
@@ -263,3 +263,4 @@ define i64 @func2() {
   %fp = call ptr @llvm.adjust.trampoline(ptr @trampg)
   ret i64 0
 }
+; CHECK-LINUX:         .section        ".note.GNU-stack","x",@progbits
```

Comment:
To quote the top of this file:

```
; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

You need `; UTC_ARGS: --disable` before this to mark a region that UTC should not be managing. Maybe also best to put an enable back after it.